aws-globalaccelerator-endpointgroup/README.md

```diff
@@ -10,6 +10,9 @@ There are a couple options for building and testing the project.
 ##### Files:
 
 1. `aws-globalaccelerator-endpointgroup.json` - This file contains the schema definition for the endpoint group resource.  It also drives auto generated classes from rpdk.
+2. `inputs/inputs_1_create.json` - This file is used during the contract tests which are run when using the "cfn submit" or "cfn test" commands.  When running either of those commands, replace {{ListenerArn}} in this file with a valid Listener Arn from one of your Global Accelerators in your aws account.
```

Comment:
nit: is it possible/easy to remove "_1_" from input file name?

Reply:
No, this is a requirement of the contract tests.  It allows for multiple versions of the same type (create, invalid, or update)

Reply:
Thanks for clarification.